.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+<!-- When opening a PR, start by forking this repository. Then, based on the type of change you're making you'll need to create a new branch from either the `master` or `live` branches:
+
+For documentation for new features, please base your fork off the master branch.
+
+If you have a typo or existing document improvement to an already shipped feature, please base your change off of the [live branch](https://github.com/MicrosoftDocs/WindowsCommunityToolkitDocs/tree/live).  This will allow us to get the change to the published documentation between releases.
+
+We will periodically merge updates from the live branch to master to keep master in-sync with the published docs.  When we make a new release, we will push master to the live branch in order to publish documentation for new features.
+
+Documentation Links
+**This link is currently only available for Microsoft Employees** - [Staging review from 'master' branch](https://review.docs.microsoft.com/windows/communitytoolkit/?branch=master)
+- [Live site from 'live' branch](https://docs.microsoft.com/windows/communitytoolkit) -->
+
+## Fixes # <!-- Link to relevant issue or Feature PR of the Windows Community Toolkit repo (for ex: #1234) which will create a reference to the associated issue and PR once it is created-->
```

Comment:
```suggestion
## Fixes # <!-- Link to a relevant issue # in this docs repository (if any, otherwise remove this line) -->
## Docs for Toolkit PR [#](https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/#) <!-- Link to relevant issue or Feature PR # of the Windows Community Toolkit repo which will create a reference to the associated issue and PR once it is created, remove if not tied to an issue or feature -->
```